static/test/javascripts-legacy/spec/commercial/third-party-tags.spec.js

```diff
@@ -9,6 +9,9 @@ define([
             tagsContainer, commercialFeatures;
 
         beforeEach(function (done) {
+            injector.mock('svgs/icon/thumb.svg', {
```

Comment:
Have you tried putting the mock into a `__mocks` directory next to the module, so we don't have to repeat this so many times?